install/mysql/glpi-empty.sql

```diff
@@ -9550,4 +9550,17 @@ CREATE TABLE `glpi_searches_criteriafilters` (
   KEY `search_itemtype` (`search_itemtype`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
 
+DROP TABLE IF EXISTS `glpi_defaultfilters`;
+CREATE TABLE `glpi_defaultfilters` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) DEFAULT NULL,
+  `is_active` tinyint NOT NULL DEFAULT '1',
+  `comment` text DEFAULT NULL,
+  `itemtype` varchar(100) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `itemtype` (`itemtype`),
```

Comment:
I want to discuss about this unicity key.
I'm not against it, but at least, a message should inform the admin when he tries to add a new filter when another one exists on the same itemtype.

Now, do we want to support only one filter per itemtype ?
Is there any technical issue having multiple ? (UI aside)